src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -33,8 +33,10 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation
     /// <summary>
     /// The base class of both the Roslyn editor factories.
     /// </summary>
-    internal abstract class AbstractEditorFactory : IVsEditorFactory, IVsEditorFactoryNotify
+    internal abstract class AbstractEditorFactory : IVsEditorFactory, IVsEditorFactory4, IVsEditorFactoryNotify
```

Comment:
IVsEditorFactory4 is being embedded, so this won't prevent loading in 16.8 will it?

Reply:
I will try it manually to verify.

Reply:
Cool. Yeah, this should be no-op on older builds.